### PR TITLE
Update keypadc.asm

### PR DESCRIPTION
**Pull request by grosged**
Optimisation of kb_ScanGroup

### CODE REVIEW

**Comment by adriweb** [comment]
(it's normal that the travis build/check will fail ; we noticed that it was passing successfully on the other PR)

I'll let other people review the change :)

**Comment by grosged** [comment]
Thanks for all, Adriweb ;) I'll now PR into llvm ;)

File: src/keypadc/keypadc.asm [hunk 1]
```diff
@@ -55,28 +55,24 @@ kb_ScanGroup:
...
-	pop	bc
-	push	bc
-	push	hl
+	ld	hl,3
+	add	hl,sp
+	ld	a,(hl)
+	cp	a,8
+	jr	nc,.found
+	add	a,a
+	add	a,$0E
```
Comment (runer112) [inline src/keypadc/keypadc.asm]: This needs to be $10.

Comment (grosged) [reply src/keypadc/keypadc.asm]: Ah yes ! You're right ;)
This "ld	de,kbdG1-2" lead me to mistake!

**Comment by grosged** [comment]
As my fork created patches here & there, I begin to get lost in branches & modified PR...
What am I supposed to do now ?.. PR into real master branch ?

**Comment by adriweb** [comment]
> As my fork created patches here & there, I begin to get lost in branches & modified PR...
> What am I supposed to do now ?.. PR into real master branch ?

You probably noticed already, but whenever you edited the file again on your fork (on the "patch-1" branch tht github created), it automatically put the new commit on this PR :) So it's all good now, and I suppose we can merge.

**Comment by runer112** [comment]
Tested and it seems to work.

**Comment by adriweb** [comment]
The 10 keypadc tests we have pass as well on travis, so merging.

**Pull request merged**